include/jemalloc/internal/tcache_inlines.h

```diff
@@ -32,13 +32,65 @@ tcache_enabled_set(tsd_t *tsd, bool enabled) {
 	tsd_slow_update(tsd);
 }
 
+static inline unsigned
+thread_nhbins_get(tsd_t *tsd) {
+	return tsd_thread_nhbins_get(tsd);
+}
+
+static inline void
+thread_nhbins_set(tsd_t * tsd, unsigned thread_nhbins) {
+	if (thread_nhbins > TCACHE_NBINS_MAX) {
+		thread_nhbins = TCACHE_NBINS_MAX;
```

Comment:
We should only have the mallctl case that can potentially have invalid inputs. Let's handle that case on the mallctl callsite, and here simply assert the value is valid.